Tests/Editor/CCUTest.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using ConditionalCompilation;
+using NUnit.Framework;
+using UnityEngine;
+
+namespace UnityEditor.Experimental.EditorVR.Tests
+{
+	[InitializeOnLoad]
+	public class CCUTest
+	{
+		[Test]
+		public void TestCCU()
+		{
+			var buildTargetGroup = EditorUserBuildSettings.selectedBuildTargetGroup;
+			Assert.IsTrue(PlayerSettings.GetScriptingDefineSymbolsForGroup(buildTargetGroup).Contains(ConditionalCompilationUtility.kEnableCCU));
+		}
+
+		[Test]
+		public void NoDependencyTest()
+		{
+			// TODO: Find a better way to collect dependencies
+			var dependencyPath = Path.Combine(EditorApplication.applicationContentsPath, "Managed");
+			var extensionsPath = Path.Combine(EditorApplication.applicationContentsPath, "UnityExtensions");
+
+			var references = new List<string>
```

Comment:
Use http://stackoverflow.com/questions/383686/how-do-you-loop-through-currently-loaded-assemblies/26300241#26300241 + http://stackoverflow.com/questions/52797/how-do-i-get-the-path-of-the-assembly-the-code-is-in